src/Microsoft.Extensions.Logging.EventLog/Internal/IEventLog.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.Extensions.Logging.EventLog.Internal
+{
+    public interface IEventLog
+    {
+        int MaxMessageSize { get; }
```

Comment:
Using huge strings is not a real problem but I think having small max size makes the test easy to read (For example: https://github.com/aspnet/Logging/pull/330/files#diff-5f8faf58806279e51869fddb0ff57828R155)